src/languages/php.js

```diff
@@ -132,11 +132,16 @@ export default function(hljs) {
         className: 'function',
         relevance: 0,
         beginKeywords: 'fn function', end: /[;{]/, excludeEnd: true,
+        keywords: 'fn function',
```

Comment:
This is implied by the previous line and should not be necessary.

Reply:
Oh, I am so sorry. I forgot to remove this line before doing a commit.